### PR TITLE
add navigation to app

### PR DESCRIPTION
**Pull request by ZeshanShahid79**
Hi I added NavLink (Navigation) to my app.

### CODE REVIEW

File: src/Pages/RestaurantCard/RestaurantCard.js [hunk 1]
```diff
@@ -34,13 +34,11 @@ export default function RestaurantCard({ entries, setAddress }) {
...
       </Button>
       <Link to="/GeoMap" onClick={() => setAddress(randomRestaurant.address)}>
-        <Button onClick={() => window.confirm('copy address to enter in Map!')}>
+        {/* <Button onClick={() => window.confirm('copy address to enter in Map!')}>
           Show Map
-        </Button>
-      </Link>
-      <Link to="/">
-        <AiOutlineArrowLeft size={60} color="white" />
+        </Button> */}
```
Comment (DanSchrader) [inline src/Pages/RestaurantCard/RestaurantCard.js]: Yeah... you know it XD

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import RestaurantCard from './Pages/RestaurantCard/RestaurantCard';
 import ContactForm from './Pages/ContactForm/ContactForm';
-import { Routes, Route } from 'react-router-dom';
+import { Routes, Route, Link } from 'react-router-dom';
```
Comment (DanSchrader) [inline src/App.js]: If you plan to style your links here and later, it's recommended to use NavLink instead of Link.

**Review by DanSchrader — COMMENTED** [review]
Cool app! Can't wait to use it!

File: src/Pages/RestaurantCard/RestaurantCard.js (hunk 1 above)
Comment (ZeshanShahid79) [reply src/Pages/RestaurantCard/RestaurantCard.js]: :D

File: src/App.js (hunk 2 above)
Comment (ZeshanShahid79) [reply src/App.js]: cheers mate :)

**Pull request merged**